internal/tiltfile/tiltfile_test.go

```diff
@@ -29,7 +29,7 @@ func TestGetServiceConfig(t *testing.T) {
 	file := tempFile(
 		fmt.Sprintf(`def blorgly():
   image = build_docker_image("%v", "docker tag", "the entrypoint")
-  image.add('/mount_points/1', local_git_repo('.'))
```

Comment:
can you add a test case for passing them in the wrong order, so we can see what the error looks like?